pkg/iplist/officeips/officeips.go

```diff
@@ -0,0 +1,85 @@
+package officeips
```

Comment:
I think this needs some slight refactoring to make sourcing IPs from an external provider more generic.   This is very implementation specific on an open source repo.

Reply:
I agree but I am not sure how far we should go. I adding some other comments on parts of this lister that I think can be abstracted from the general flow of a 'ip client'. These are true of the github lister also.